modules/system/lang/fr/lang.php

```diff
@@ -281,13 +285,18 @@
         'plugin_version' => 'Version',
         'plugin_author' => 'Auteur',
         'plugin_not_found' => 'Plugin non trouvé',
+        'plugin_version_not_found' => 'Version du plugin non trouvée',
         'core_current_build' => 'Version actuelle',
         'core_view_changelog' => 'Voir le Changelog',
         'core_build' => 'Version :build',
         'core_build_help' => 'Une nouvelle version est disponible.',
         'core_downloading' => 'Téléchargement des fichiers de l’application',
         'core_extracting' => 'Décompression des fichiers de l’application',
         'core_set_build' => 'Définition du numéro de build',
+        'update_warnings_title' => 'Certains problèmes ont été détectés et nécessitent une attention particulière :',
+        'update_warnings_plugin_missing' => 'Le plugin :parent_code nécessite l\'installation de :code pour fonctionner.',
+        'update_warnings_plugin_replace' => 'Le plugin :plugin remplace :alias, veuillez supprimer :alias pour vous assurer qu\'il n\'y ait pas de conflit.',
+        'update_warnings_plugin_replace_cli' => 'Ce plugin remplace :alias, veuillez supprimer :alias pour vous assurer qu\'il n\'y ait pas de conflits.',
```

Comment:
Can wrap in double quotes for lines with lots of single quotes so you don't have to escape every single quote